entrypoint.d/20-postgres-wait

```diff
@@ -6,6 +6,12 @@ fi
 
 log INFO Waiting until postgres is listening at $PGHOST...
 while true; do
-    psql --list > /dev/null 2>&1 && break
+    if [ -n "$PGDATABASE" ]; then
+        echo "SELECT 1;" | psql "$PGDATABASE"
```

Comment:
this breaks waiting for postgres container when $PGDATABASE is defined and the database would be created by Odoo but the database does not exist yet when entrypoint.d/20-postgres-wait is running.

Reply:
Good point, what about a simple `or` between the two ? Would that fix this issue for you ?
For the case I described, this would allow to pass this script safely, and for your case also, while making sure we checked the access (HOST/PORT/USER/PASSWORD) to postgres...

Reply:
yes an or should work, as the (error) output is redirected to /dev/null we do not loose any information.